### PR TITLE
Sync telemetry and user added to etl

### PR DESCRIPTION
**Pull request by ak2502**

### CODE REVIEW

File: src/main/java/org/avniproject/etl/repository/rowMappers/tableMappers/UserTable.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+public class UserTable extends Table{
+    @Override
+    public String name(Map<String, Object> tableDetails) {
+        return "user";
+    }
+
+    @Override
+    public List<Column> columns() {
+        return new Columns()
+                .withSerialIdColumn()
```
Comment (vinayvenu) [inline src/main/java/org/avniproject/etl/repository/rowMappers/tableMappers/UserTable.java:17]: we don't need a serial Id column. The id can be the same as the one in users.

File: src/main/resources/sql/etl/user.sql.st [hunk 2]
```diff
@@ -0,0 +1,21 @@
+INSERT INTO <schemaName>.<tableName> (id, uuid, username, organisation_id, created_by_id,
+    last_modified_by_id, created_date_time, last_modified_date_time, catchment_id,
+   is_org_admin, email, phone_number, name)
```
Comment (vinayvenu) [inline src/main/resources/sql/etl/user.sql.st:3]: is_org_admin is no longer relevant, and can be removed.

**Review by vinayvenu — CHANGES_REQUESTED** [review]
Take a look at the DuplicateRowDeleteAction. This will create duplicates when the last_modified_date_time for a row changes. We will need to delete them as well.

**Review by vinayvenu — APPROVED** [review]

File: .java-version [hunk 3]
```diff
@@ -1 +1 @@
-17
+openjdk64-17.0.7
```
Comment (vinayvenu) [inline .java-version]: I am guessing this is removed later

File: src/main/resources/sql/etl/deleteDuplicateSyncTelemetry.sql.st [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+                    sync_end_time,
+                    device_name,
+                    android_version,
+                    app_version,
+                    device_info,
+                    sync_source
+FROM "<schemaName>"."sync_telemetry";
+DROP TABLE "<schemaName>"."sync_telemetry";
+ALTER TABLE "<schemaName>"."temp_table"  
+RENAME TO "sync_telemetry";
```
Comment (vinayvenu) [inline src/main/resources/sql/etl/deleteDuplicateSyncTelemetry.sql.st:27]: We cannot recreate sync_telemetry table every time ETL runs. We have to do an incremental change here.

**Review by vinayvenu — CHANGES_REQUESTED** [review]
Other than comments on lines, tests will need to be added.

**Review by vinayvenu — CHANGES_REQUESTED** [review]
Duplicate rows will need to be handled. It is not done right now. 
Please double check if we have a serial id or a regular id column

File: .java-version (hunk 3 above)
Comment (ak2502) [reply .java-version]: yes. It is removed

File: src/main/resources/sql/etl/deleteDuplicateSyncTelemetry.sql.st (hunk 4 above)
Comment (ak2502) [reply src/main/resources/sql/etl/deleteDuplicateSyncTelemetry.sql.st:27]: this is not being used. I think file got pushed by mistake

Comment (ak2502) [reply src/main/resources/sql/etl/deleteDuplicateSyncTelemetry.sql.st:27]: this is not being used. I think file got pushed by mistake

**Pull request closed without merging**